libr/core/cbin.c

```diff
@@ -403,6 +403,7 @@ static bool bin_raw_strings(RCore *r, int mode, int va) {
 	}
 	RList *l = r_bin_raw_strings (bf, 0);
 	_print_strings (r, l, mode, va);
+	r_list_free(l);
```

Comment:
missing space before (

Reply:
Wasn't `git diff master..mybranch | ./sys/clang-format-diff.py -p1` supposed to catch that? It detected at least one other case.

Reply:
Done.